src/BigANNBinaryEmbeddingScenarioBase.cs

```diff
@@ -63,6 +63,31 @@ protected string GetQueryDataPath()
 
         protected override ContainerProperties GetContainerSpec(string containerName)
         {
+
+            bool enableShardedDiskAnn = Convert.ToBoolean(this.Configurations["AppSettings:scenario:enableShardedDiskANN"]);
+            ;
+            VectorIndexPath vectorIndexPath;
+            if (enableShardedDiskAnn)
+            {
+                string shardKey = this.Configurations["AppSettings:scenario:vectorIndexShardKey"]  ?? throw new ArgumentNullException("AppSettings:scenario:vectorIndexShardKey");
+                vectorIndexPath = new VectorIndexPath()
+                {
+                    Path = this.EmbeddingPath,
+                    Type = VectorIndexType.DiskANN,
```

Comment:
anyway to not repeat :
                   Path = this.EmbeddingPath,
                    Type = VectorIndexType.DiskANN,